nvm.sh

```diff
@@ -2357,6 +2357,9 @@ nvm() {
         setopt shwordsplit
       fi
       nvm_err "nvm --version: v$(nvm --version)"
+      if [ -n "$TERM_PROGRAM" ]; then
```

Comment:
hmmm ... didn't know that before, could learn something here, I'll figure it out :) Fixed now.